### DISCUSSION TITLE
WebGIS6 - Editieren - Validierungsfehlermeldung

### DISCUSSION THREAD

**Discussion by Andreas-Lampert** [e-netze/webgis-community#209] [Q&A]
Hallo Jürgen!

Im WebGIS5 wird beim Eintragen eines ungültigen Wertes und einer konfigurierten Validierungsfehlermeldung die Meldung zweizeilig dargestellt.
![image](https://github.com/user-attachments/assets/26b5ff8a-c1a1-46c1-9065-b6e0ded9d25e)

Im WebGIS6 (6.24.4002) ist die Meldung nur einzeilig und rechtsbündig. Damit ist der Beginn der Meldung nicht lesbar. 
![image](https://github.com/user-attachments/assets/5f7d377b-d6b7-46b5-854c-27730b354d9a)

Außerdem wird die Meldung im WebGIS6 immer dargestellt, egal ob schon ein Wert eingetragen wurde oder nicht. Im WebGIS5 wird die Meldung erst angezeigt, wenn ein falscher Wert angegeben wurde. Ist das Verhalten im WebGIS6 so geplant?

Könntest du in der GUI diese Meldung wieder linksbündig und ggf. zweizeilig machen.
Danke!

Lg, Andi

**Comment by gis-eni** [comment]
Wurde verbessert (vergrößert sich beim Klick).

Ansonsten kann die ``.webgis-validation-error.haserror`` in der (custom) default.css geändert werden
